lib/rbac/policies.rb

```diff
@@ -18,7 +18,7 @@ def add_policy(name, role_uuid)
     # delete all policies that contains the role.
     def delete_policy(role)
       RBAC::Service.call(RBACApiClient::PolicyApi) do |api_instance|
-        RBAC::Service.paginate(api_instance, :list_policies, :name => @prefix).each do |policy|
+        RBAC::Service.paginate(api_instance, :list_policies, :name => @prefix, :limit => 500).each do |policy|
```

Comment:
@hsong-rh Should this be using the DEFAULT_LIMIT that is being defined in the lib/rbac/access.rb?